css/css-properties-values-api/META.yml

```diff
@@ -872,13 +873,29 @@ links:
           subtest: CSSOM setters function as expected for unregistered properties
         - test: registered-property-cssom.html
           subtest: Initially unregistered values from stylesheet
+        - test: at-property-cssom.html
```

Comment:
Need these

Reply:
update: just merged in https://github.com/web-platform-tests/wpt-metadata/pull/3464

This is now covered